src/toGamut.js

```diff
@@ -189,25 +189,18 @@ export function toGamutCSS (origin, { space = origin.space }) {
 		current.coords[1] = chroma;
 		if (min_inGamut && inGamut(current, space)) {
 			min = chroma;
-			continue;
 		}
 		else if (!inGamut(current, space)) {
 			const clipped = clip(current);
 			const E = deltaEOK(clipped, current);
+			// Note- this is missing several steps of the CSS Gamut Mapping
+			// Algorithm at this point.
 			if (E < JND) {
-				if ((JND - E < ε)) {
-					// match found
-					current = clipped;
-					break;
-				}
-				else {
-					min_inGamut = false;
```

Comment:
I don't understand why those two lines are removed

Reply:
There's some more context and examples in the description of this issue, but this resolves an issue I was seeing (although I'm not confident in this correct resolution.)

For some colors the `if ((JND - E < ε))` branch was never true, and so the algorithm continued until the `while` conditional failed, at which point the color was back out of gamut.

Reply:
Ah, I see